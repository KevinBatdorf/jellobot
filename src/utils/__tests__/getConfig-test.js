const { processConfig } = require('../getConfig');

it('works', () => {
  const config = processConfig({ nick: 'jellobot' });
  expect(config).toMatchSnapshot();
});
