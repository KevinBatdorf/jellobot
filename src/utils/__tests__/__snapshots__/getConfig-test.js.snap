// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
Object {
  "channels": Array [
    Object {
      "name": "##jellobot-test",
    },
  ],
  "commandPrefix": "!",
  "ircClientConfig": Object {
    "channels": Array [
      "##jellobot-test",
    ],
    "retryCount": 10,
  },
  "nick": "jellobot",
  "password": null,
  "plugins": Object {},
  "server": "irc.freenode.net",
  "verbose": false,
}
`;
